css/style.css

```diff
@@ -0,0 +1,462 @@
+  border: none;
+  border-radius: 10px;
+  color: #fff;
+  font-family: Roboto;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 50px;
+  height: 90px;
+  width: 350px;
+  outline: none; 
+}
+
+.start-button:hover {
+  cursor: pointer;
+  opacity: .9;
+}
+
+button::active,
+button::focus {
+  outline: none !important;
+}
+
+button::-moz-focus-inner {
+  border: 0 !important;
+}
+
+.pictures {
+  position: absolute;
+  bottom: 8%;
+  right: 8%;
+}
+
+.bottom-card {
+  background-image: url(../img/card.png);
+  height: 341px;
+  width: 218px;
+  transform: rotate(11.9deg);
+}
+
+.top-card {
+  background-image: url(../img/card.png);
+  height: 341px;
+  width: 218px;
+  position: relative;
+  right: 45px;
+  top: 20px;
+  transform: rotate(39.26deg);
+}
+
+/* Стили, которые добавляются через скрипт */
+
+.card-wrap {
+  box-shadow: 1px 1px 2px rgba(0, 0, 0, .7);
+  cursor: pointer;
+  height: 340px;
+  width: 220px;
+  position: relative;
+  transform-style: preserve-3d;
+  transition: transform .8s;
+}
+
+.card-wrap__hover:hover {
+  box-shadow: 10px 10px 15px rgba(0, 0, 0, .5);
+  transform: rotate(-3.08deg) translateY(-15px);
+  transition: .5s ease;
+}
+
+.card__click {
+  transform: rotateY(180deg);
+}
+
+.card__back {
+  background: url(../img/card.png) no-repeat;
+  backface-visibility: hidden;
+  height: 100%;
+  width: 100%;
+  position: absolute;
+  -webkit-backface-visibility: hidden;
+}
+
+.card__game-over {
+  background: url(../img/game_over.png) no-repeat;
+  backface-visibility: hidden;
+  height: 100%;
+  width: 100%;
+  position: absolute;
+  transform: rotateY(180deg);
+  -webkit-backface-visibility: hidden;
+}
+
+.card__winner {
+  background: url(../img/bug_card.png) no-repeat;
+  backface-visibility: hidden;
+  height: 100%;
+  width: 100%;
+  position: absolute;
+  transform: rotateY(180deg);
+  -webkit-backface-visibility: hidden;
+}
+
+
+.three-card {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  position: relative;
+  height: 340px;
+  width: 770px;
+  margin: auto;
+  perspective: 1000px;
+}
+
+.six-card {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  flex-wrap: wrap;
+  height: 750px;
+  width: 770px;
+  margin: auto;
+  perspective: 1000px;
+}
+
+.ten-card {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  flex-wrap: wrap;
+  height: 750px;
+  width: 1180px;
+  margin: auto;
+  perspective: 1000px;
+}
+
+@media (min-width: 1025px) and (max-width: 1224px) {
+
+  .menu { margin: 0 0 0 70px; }
+
+  .level_btn {
+    font-size: 40px;
+    line-height: 50px;
+  }
+
+  .ten-card .card-wrap,
+  .ten-card .card__back,
+  .ten-card .card__game-over,
+  .ten-card .card__winner {
+    background-size: 175px 270px;
+    height: 270px;
+    width: 175px;
+  }
+
+  .ten-card {
+    height: 570px;
+    width: 935px;
+  }
+
+}
+
+@media (min-width: 987px) and (max-width: 1024px) {
+  
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Olga_FinalGame_JS/pullRequest?prid=5839198)